CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # Changelog
 
+## HEAD
```

Comment:
This one should probably be `v0.15.0-dev` to be in line with the history of the file.